<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TufeiListItem</class>
 <widget class="QWidget" name="TufeiListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_tufeiLevelVal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tufeiBackGround">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tufeiHpValue">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0/0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tufeiName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tufeiLevel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>level</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tufeiIcon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tufeiHp">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tufeiFinger">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lb_tufeiBackGround</zorder>
  <zorder>lb_tufeiHpValue</zorder>
  <zorder>lb_tufeiName</zorder>
  <zorder>lb_tufeiLevel</zorder>
  <zorder>lb_tufeiIcon</zorder>
  <zorder>lb_tufeiHp</zorder>
  <zorder>lb_tufeiFinger</zorder>
  <zorder>lb_tufeiLevelVal</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
